### ISSUE TITLE
Page structure in admin

### ISSUE THREAD

**Issue by yunmanger1** [divio/django-cms#5281]
I having an issue with buggy custom plugins added to placeholders. For example I add some plugin that fails to render because of wrong user configuration, the whole page gets HTTP 500 and there is no way to edit it.

Is it possible somehow remove the plugin without digging into shell? I think the intuitive solution would be to have the page structure available in admin. So that it can be editable without rendering.

**Comment by evildmp** [comment]
I agree that a good way to handle this would be to place a direct link in the Admin > CMS > Pages list, for each page, that would open the page in its structure mode, bypassing the page rendering.

In the case of a broken plugin, this could save a trip to the shell to remove it.

@FinalAngel @vxsx what do you think? We don't want to overload the page tree with icons; perhaps the View icon could incorporate this option.

**Comment by FinalAngel** [comment]
@evildmp @yunmanger1 we're looking into solutions for this when working on splitting content form structure further.

**Comment by Aiky30** [comment]
Do not close this issue, this same issue is raised many times. I will collate them and concentrate on one issue for this task.
